sams/sampler/Core.py

```diff
@@ -48,6 +48,7 @@ def init(self):
             "jobid": self.config.get(["options", "jobid"]),
             "node": self.config.get(["options", "node"]),
         }
+        self._most_recent_sample = self._storage_wrapping(self.core)
```

Comment:
```suggestion
        self._most_recent_sample = [self._storage_wrapping(self.core)]
```

I have thought a bit about it and it will simplify interaction if most_recent_sample is always a list, which matches the fact that storage is pushed into a queue (so it is the set of pushes done by the last call to sample()).